View Controllers/ListViewController.m

```diff
@@ -12,6 +12,10 @@
 #import "HMSegmentedControl.h"
 #import "ProfileListCell.h"
 #import "DetailsViewController.h"
+#import "NCHelper.h"
+
+static NSString* NO_FAVORITE_MSG = @"You have no favorites!";
```

Comment:
yesss exactly the right idea! the practice here tends to be "static const NSString *k...."... starting the variable name with a k is a super old practice but lots of ios tend to do it. 

here are a lot of examples from our facebook codebase: https://fburl.com/biggrep/u8traxah